src/js/gi/components/search/SearchResult.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../../actions';
+
+import { Link } from 'react-router';
+import { estimatedBenefits } from '../../selectors/estimator';
+
+export class SearchResult extends React.Component {
+
+  cautionFlag() {
+    if (!this.props.cautionFlag) {
+      return null;
+    }
+    return (
+      <div className="caution-flag">
+        <i className="fa fa-warning"></i>
+        Caution
+      </div>
+    );
+  }
+
+  place() {
+    const domestic = <span>{this.props.city}, {this.props.state}</span>;
+    const foreign = <span>{this.props.city}, {this.props.country}</span>;
+    return (this.props.country === 'usa' ? domestic : foreign);
+  }
+
+  renderEstimate({ qualifier, value }) {
+    const formatCurrency = (n) => {
+      const str = Math.round(Number(n)).toString();
+      return str.replace(/\d(?=(\d{3})+$)/g, '$&,');
+    };
+    if (qualifier === '% of instate tuition') {
+      return <span>{value}% in-state</span>;
+    }
+    if (qualifier === null) {
+      if (value === 'N/A') {
+        return 'N/A';
+      }
+      return value;
+    }
+    return (<span>${formatCurrency(value)}</span>);
+  }
+
+  render() {
+    const tuition = this.renderEstimate(this.props.estimated.tuition);
+    const housing = this.renderEstimate(this.props.estimated.housing);
+    const books = this.renderEstimate(this.props.estimated.books);
+
+    return (
+      <div className="search-result">
+        <div className="outer">
+          {this.cautionFlag.bind(this)()}
```

Comment:
Usually we just bind every method in the constructor and don't have to deal with fixing references to `this` later.

```
constructor(props) {
  super(props);
  this.cautionFlag = this.cautionFlag.bind(this);
  this.place = this.place.bind(this);
  this.renderEstimate = this.renderEstimate.bind(this);
}